docsite/rst/galaxy.rst

```diff
@@ -129,48 +128,100 @@ And here's an example showing some specific version downloads from multiple sour
       scm: git
       version: 0.1.0
 
-As you can see in the above, there are a large amount of controls available
-to customize where roles can be pulled from, and what to save roles as. 
+Installing multiple roles from multiple files
+=============================================
+
```

Comment:
It looks to me like it is implemented: https://github.com/ansible/ansible/blob/devel/lib/ansible/cli/galaxy.py#L358

I have not actually tested, so maybe it's not fully implemented?